sandbox/load_test_jobs.py

```
@@ -701,6 +761,137 @@ def build(_idx: int) -> Shell:
+                "beauty",
+                command=["/bin/echo", "rendered", "frame", "#IFRAME#"],
+                range="1-%d" % nframes,
+            ))
+            # Always paused: the marker frames would otherwise finish instantly
+            # and the job would leave the active list before you can open it.
+            outline.cuerun.launch(ol, pause=True, use_pycuerun=False)
+
+            # Poll for visibility rather than a one-shot lookup: _find_job can
+            # race Cuebot right after launch and would skip registerOutputPath.
+            job = _wait_for_jobs([short_name])[0]
+            layer = next(l for l in job.getLayers() if l.name() == "beauty")
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)